src/test/kotlin/fr/gouv/agora/usecase/reponseConsultation/InsertReponseConsultationUseCaseTest.kt

```diff
@@ -186,10 +187,13 @@ internal class InsertReponseConsultationUseCaseTest {
         // Then
         assertThat(result).isEqualTo(InsertResult.INSERT_SUCCESS)
         then(consultationInfoRepository).should(only()).getConsultation(consultationId = "consultId")
-        then(consultationResponseRepository).should(only()).hasAnsweredConsultation(
+        then(userAnsweredConsultationRepository).should(times(1)).hasAnsweredConsultation(
             consultationId = "consultId",
             userId = "userId",
         )
+        then(userAnsweredConsultationRepository).should(times(1)).insertUserAnsweredConsultation(
+            UserAnsweredConsultation(userId = "userId", consultationId = "consultId")
+        )
```

Comment:
Enlever les `times(1)`, c'est identique à `should()` sans arguments
Par contre il faut ajouter un `then(userAnsweredConsultationRepository).shouldHaveNoMoreInteractions()` pour vérifier qu'il ne fait pas d'autres appels derrière

Reply:
done